### PR TITLE
MAT-6610 reorder Qdm strat population values

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
## MADiE PR

Jira Ticket: [MAT-6610](https://jira.cms.gov/browse/MAT-6610)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [x] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/main/java/cms/gov/madie/measure/utils/JsonUtil.java [hunk 1]
```diff
@@ -391,30 +390,13 @@ private static void handlePopulationValues(
...
-    if (population.get("NUMER") != null) {
-      TestCasePopulationValue populationValue =
-          TestCasePopulationValue.builder()
-              .name(PopulationType.NUMERATOR)
-              .expected(population.get("NUMER").asInt())
-              .build();
-      populationValues.add(populationValue);
-    }
+
+    fetchObservationValuesForCV(population, populationValues, measure);
```
Comment (nmorasb) [inline src/main/java/cms/gov/madie/measure/utils/JsonUtil.java]: can we rename this function to indicate a little better that it is updating input parameters (fetch implies it's grabbing some data, not making updates to input parameters)

File: src/main/java/cms/gov/madie/measure/utils/JsonUtil.java [hunk 2]
```diff
@@ -391,30 +390,13 @@ private static void handlePopulationValues(
...
+
+    fetchDenominatorValues(population, populationValues, measure);
```
Comment (nmorasb) [inline src/main/java/cms/gov/madie/measure/utils/JsonUtil.java]: same as above - can we please rename this function

File: src/main/java/cms/gov/madie/measure/utils/JsonUtil.java [hunk 3]
```diff
@@ -391,30 +390,13 @@ private static void handlePopulationValues(
...
+
+    fetchNumeratorValues(population, populationValues, measure);
```
Comment (nmorasb) [inline src/main/java/cms/gov/madie/measure/utils/JsonUtil.java]: same as above - can we please rename this function

File: src/main/java/cms/gov/madie/measure/utils/JsonUtil.java (hunk 1 above)
Comment (sb-cecilialiu) [reply src/main/java/cms/gov/madie/measure/utils/JsonUtil.java]: @nmorasb would updateTestCasePopulationObservationValuesForCV be ok?

File: src/main/java/cms/gov/madie/measure/utils/JsonUtil.java (hunk 2 above)
Comment (sb-cecilialiu) [reply src/main/java/cms/gov/madie/measure/utils/JsonUtil.java]: @nmorasb updateTestCasePopulationDenominatorValues?

File: src/main/java/cms/gov/madie/measure/utils/JsonUtil.java (hunk 3 above)
Comment (sb-cecilialiu) [reply src/main/java/cms/gov/madie/measure/utils/JsonUtil.java]: @nmorasb updateTestCasePopulationNumeratorValues?

File: src/main/java/cms/gov/madie/measure/utils/JsonUtil.java (hunk 1 above)
Comment (sb-cecilialiu) [reply src/main/java/cms/gov/madie/measure/utils/JsonUtil.java]: Talked to Nate, will use setObservationValuesForCV

Comment (sb-cecilialiu) [reply src/main/java/cms/gov/madie/measure/utils/JsonUtil.java]: @nmorasb changed to suggested names, including the ones below.

**Review by nmorasb — APPROVED** [review]

**Pull request merged**